connectivity/FEATURE_BLE/source/cordio/source/PalAttClientImpl.h

```diff
@@ -173,8 +173,10 @@ class PalAttClient final : public ble::PalAttClient {
      */
     ble_error_t terminate() final;
 
+#if BLE_FEATURE_GATT_CLIENT
```

Comment:
or server?

Reply:
Why would you need the GATT client for if it hasn't been selected ?

Reply:
yes, sorry, Friday evening